group_vars/all

```diff
@@ -53,3 +53,21 @@ ansible_git_repository: git://github.com/buildbot/buildbot-infra
 
 # syslog server to which all syslog messages should be forwarded
 global_syslog_server: 192.168.80.51
+
+# `hosts` declares ip addresses for _all_ machines and jails under Ansible
+# management.
+hosts:
+  ns1:
+    vlan0: "192.168.80.236"
```

Comment:
That's why I suggested "purpose" instead of "interface".  I don't think we
want to hard-code that 'vlan0' is always external, for example.